language/move-vm/runtime/Cargo.toml

```diff
@@ -12,31 +12,32 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-better_any = "0.1.1"
-fail = "0.4.0"
-once_cell = "1.7.2"
-parking_lot = "0.11.1"
-sha3 = "0.9.1"
-tracing = "0.1.26"
+better_any = { version = "0.1", default-features = false }
+fail = { version = "0.5", default-features = false, optional = true }
+sha3 = { version = "0.10", default-features = false }
+tracing = { version = "0.1", default-features = false }
+hashbrown = { version = "0.14", default-features = false, features = ["ahash"] }
+once_cell = { version = "1.7", default-features = false, optional = true }
```

Comment:
There is a 1.18 version of the crate - could we update it?

Reply:
ah! missed that one - good catch!

Done